test/unit/mocks.js

```diff
@@ -1,13 +1,49 @@
 define(["ObjectMakr"], function (ObjectMakrModule) {
     var ObjectMakr = ObjectMakrModule.ObjectMakr;
+    var expect = require("chai").expect;
 
     var mocks = {
         /**
-         * 
+         * Returns an instance of ObjectMakr.
          */
         mockObjectMakr: function (settings) {
-            return new ObjectMakr(unitsize)
-        }
+            return new ObjectMakr(settings = {
```

Comment:
As a default parameter, so I don't have to have the settings object in all the tests in the `Arrange` section.
If you want me to put the settings in all the tests, I can do that once this is ready to be merged.